docs/development.rst

```diff
@@ -0,0 +1,151 @@
+
+    $ make requirements
+
+
+Run tests
+^^^^^^^^^
+
+This application uses `tox`_ to run tests under multiple python and Django version. In addition, the following test
+utilities are used:
+
+* `pytest`_ framework and test runner to run test
+* `pytest-cov`_ plugin for collecting test coverage
+* `diff-cover`_ for analyzing changeset coverage
+
+.. _tox: https://tox.readthedocs.io/en/latest/
+.. _pytest: http://doc.pytest.org/en/latest/
+.. _pytest-cov: https://pypi.python.org/pypi/pytest-cov
+.. _diff-cover: https://github.com/Bachmann1234/diff-cover
+
+`edx-enterprise` Makefile provide multiple shortcuts to running tests, the most noteable are:
+
+.. code-block:: bash
+
+    $ make test                     # run all tests in current virtualenv
+    $ make diff_cover               # run all tests in current virtualenv and report diff coverage
+
+    # run quality checks, all tests in every supported env (python and django versions) and jasmine JS tests
+    $ make test-all
+
+More info about running tests is available in :ref:`tests-section` section.
+
+
+Check quality
+^^^^^^^^^^^^^
+
+A whole bunch of static analyzers are used to ensure code quality. Most notable are ``pycodestyle`` (formerly ``pep8``),
+``pylint`` and ``jshint``.
+
+.. code-block:: bash
+
+    $ make quality      # runs all python and docs quality check
+    $ tox -e quality    # same
+    $ make jshint       # runs jshint on files in enterprise folder
+
+Build (these) docs
+^^^^^^^^^^^^^^^^^^
+
+This documentation is generated by `Sphinx`_. Essentially, there are two parts of documentation - manually written
+(located in ``docs`` folder) and generated from docstrings.
+
+.. _Sphinx: http://www.sphinx-doc.org/en/stable/
+
+.. code-block:: bash
+
+    $ tox -e docs   # performs docs quality check and generates docs in HTML
+    $ make docs     # same, but also opens browser at the index page
+
+
+Update translations
+^^^^^^^^^^^^^^^^^^^
+
+See :ref:`internationalization-section` chapter for details.
+
+Upgrading local setup from older versions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you're migrating from an older version (i.e. pre Nov 2016) of edx-platform, you might need to ensure edx-enterprise
+is installed correctly. Three things need to happen:
+
+1. ``edx-enterprise`` must be installed in edxapp env.
+2. ``edx-enterprise`` must be added to ``INSTALLED_APPS``.
+3. Migrations need to be run.
+
+All three should happen automatically if you use paver commands to upgrade your setup, but just in case something goes
+wrong with the setup, here're instructions to manually perform the upgrade.
+
+First, install ``edx-enterprise`` into virtualenv. In ``edxapp`` virtualenv (``$current_release`` is |release|)\ [#f1]_
+
+.. code-block:: bash
+
+    $ cd /edx/app/edxapp/edx-platform
+    $ pip install edx-enterprise==$current_release
+
+
+Than, make sure ``edx-enterprise`` to ``INSTALLED_APPS`` or ``OPTIONAL_APPS`` (see `lms/env/common.py`_ as an example)
```

Comment:
"Then, make sure" . There's a verb missing here. Is it "make sure ``edx-enterprise`` exists in ``INSTALLED_APPS``" or "is included in"?